src/transformers/models/data2vec/modeling_tf_data2vec_vision.py

```diff
@@ -910,6 +910,17 @@ def call(
 
         return outputs
 
+    def serving_output(self, output: TFData2VecVisionModelOutputWithPooling) -> TFData2VecVisionModelOutputWithPooling:
+        hs = tf.convert_to_tensor(output.hidden_states) if self.config.output_hidden_states else None
```

Comment:
Not a big fan of those two names, but even less fan of having the code span several lines. So if `hiddens` and `attentions`  still fit in one line, please use those (here and in the others) otherwise feel free to ignore.

Reply:
Agreed - updated across the model files this PR touches 👍